token/jwt/claims_id_token_test.go

```diff
@@ -66,7 +66,6 @@ func TestIDTokenClaimsToMap(t *testing.T) {
 		"rat":       float64(idTokenClaims.RequestedAt.Unix()),
 		"iss":       idTokenClaims.Issuer,
 		"aud":       idTokenClaims.Audience,
-		"nonce":     idTokenClaims.Nonce,
```

Comment:
Could you please add a test which checks if the nonce is set when it's set and vice versa? This basically removes the original test behaviour and just replaces it with the new one. A regression (forgot to set nonce) would not be detected with this.

Reply:
Done.